src/core/server/base.js

```diff
@@ -208,6 +208,9 @@ function baseServer(
     app.use(middleware.serveAssetsLocally());
   }
 
+  // Cookies.
```

Comment:
I don't think this comment adds anything. The code already explains it.

However, it might help to add a comment saying how this will patch `universalCookies` on to `request` since that part is a bit magical.